src/app/modules/encrpytion.ts

```diff
@@ -0,0 +1,21 @@
+import bcrypt from "bcryptjs"
+
+const SALT_WORK_FACTOR = 10 // This determines how secure the salt should be
+
+export const encryptPassword = (password) => {
+  const salt = bcrypt.genSaltSync(SALT_WORK_FACTOR) // This generates a random salt
```

Comment:
bcrypt is a popular salt and hashing dependency. It does all the heavy lifting of scrambling passwords and then unscrambling later when it's time to fetch. 

We still need session middleware for cookies and actual login before this is done